test/integration/cli-nfs-shared-volumes-create-failure.test.js

```diff
@@ -0,0 +1,149 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Copyright (c) 2017, Joyent, Inc.
+ */
+
+//
+// Summary:
+//
+// These tests ensure that when provisioning fails for a volapi storage VM, we
+// get an error message from sdc-docker rather than a message telling us the
+// creation was successful. We "break" provisioning for the 10g package by
```

Comment:
Just thinking out loud, not something I think we can fix now: I don't like the fact that this changes the ability to provision volumes while the test is running, but it doesn't really make things worse now. It would be a problem if e.g we ran some tests in parallel with other tests.

I have in my todo list to "create ticket for sdc-vmapi to be able to pass a flag that makes a provisioning workflow fail", but then we'd need to pass that flag from sdc-docker, not sure how sound that would be.

But again, this is just thinking out loud, nothing we can act on now, and I know you're already aware of this.